File: cpp/a006.cpp
#include <iostream>
#include <math.h>
using namespace std;;


int main()
{
	int a, b, c,solve1,solve2;
	cin >> a >> b >> c;
	solve1 = (-b + sqrt(b * b - 4 * a * c))/(2*a);
	solve2 = (-b - sqrt(b * b - 4 * a * c))/(2*a);
	if (b * b - 4 * a * c > 0) {
		cout << "Two different roots x1="<<solve1<<" , x2="<<solve2<<'\n';
	}
	else if (b * b - 4 * a * c == 0) {
		cout << "Two same roots x="<<solve1<<'\n';
	}
	else {
		cout << "No real root" << '\n';
	}
}

File: python/a006.py
a,b,c = map(int,input().split())

if b*b - 4 * (a*c) > 0  :
    s1 = int((-b + (b*b-(4*a*c))**0.5) // (2*a))
    s2 = int((-b - (b*b-(4*a*c))**0.5) // (2*a))
    big = max(s1,s2)
    small = min(s1,s2)
    print(f"Two different roots x1={big} , x2={small}")
elif b*b - 4* (a*c) == 0:
    s1 = int((-b + (b*b-(4*a*c))**0.5) // (2*a))
    print(f"Two same roots x={s1}")
else:
    print("No real root")